lib/bundler/cli/inject.rb

```diff
@@ -31,7 +31,7 @@ def run
 
       if added.any?
         Bundler.ui.confirm "Added to Gemfile:"
-        Bundler.ui.confirm added.map {|g| "  #{g}" }.join("\n")
+        Bundler.ui.confirm added.map {|g| "  #{g}, group => #{g.groups.inspect}, :source => '#{g.source}'" }.join("\n")
```

Comment:
I didn't understand what this means ? @segiddins,  Can you tell me what should be replaced.